src/UploadedFile.php

```diff
@@ -0,0 +1,310 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
```

Comment:
Yeah sorry, started with [that one](https://github.com/zendframework/zend-diactoros/blob/master/src/UploadedFile.php), but then heavily refactored as you can see